example/cpputest-unittests/minimal/tests/README.md

```diff
@@ -0,0 +1,36 @@
+# Embedded unit tests!
```

Comment:
might be able to merge the two readmes you have up a couple levels but this is fine too